.github/workflows/ci.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
           python -m pip install --upgrade pip setuptools
           python -m pip install PyYAML argparse rospkg vcstools catkin_pkg python-dateutil rosdistro
           python -m pip install -e .
-          python -m pip install pytest pytest-cov flake8 mock codecov
+          python -m pip install pytest pytest-cov flake8 flake8-comprehensions mock codecov
```

Comment:
This package hasn't been updated to use extras_require but I'll open a PR for that once this is in to avoid further conflicts.